programmers/두-정수사이의-합.js

```diff
@@ -0,0 +1,9 @@
+function solution(a, b) {
+    let MAX = Math.max(a, b);
+    let min = Math.min(a, b);
+    let sum = 0;
+    for(let i = min ; i <= MAX ; i+=1) {
+        sum += i;
+    }
+    return sum;
+}
```

Comment:
```suggestion
function solution(a, b) {
    const maxNum = Math.max(a, b);
    const minNum = Math.min(a, b);
    let sum = 0;
    
    for (let i = minNum; i < maxNum + 1; i += 1) {
        sum += i;
    }
    return sum;
}
```
변수명이 센스있는 것처럼 보이려고 하신거같은데 저러지마세요
let 보다는 const를 사용하고
변수선언이 끝나면 줄바꿈 한번 해주시고 for, if 은 뒤에 괄호 시작과 공백
기호화 문자 숫자 사이도 공백, 세미콜론은 공백 X
for문에서 조건식은 <= 보다는 < 가 좋다고 생각합니다.

이 문제 숙제는 제가 < 가 좋다고 생각한 이유를 알아와주세요

```js
function solution(a, b) {
    let sum = 0;
    
    for (let i = Math.min(a, b) ; i < Math.max(a, b) + 1 ; i += 1) {
        sum += i;
    }
    return sum;
}
```
이건 혹시나 신기해 할까봐